src/lib/styles/stylesheet-processor.ts

```diff
@@ -42,10 +41,25 @@ export class StylesheetProcessor {
       browserslistData: this.browserslistData,
     };
 
-    const { css, warnings } = this.styleSheetProcessorWorker(workerOptions);
+    const ioChannel = new MessageChannel();
 
-    warnings.forEach(msg => log.warn(msg));
+    try {
+    const signal = new Int32Array(new SharedArrayBuffer(4));
```

Comment:
Indentation seems off here

Reply:
Umm... strange the commit hook should have formatted this.